### PR TITLE
Remove LTI assignment selection

### PR DESCRIPTION
**Pull request by jessouyangbraven**
### Description

**Task**: https://app.asana.com/0/1174274412967132/1199394675251953

### Motivation and Context

We no longer use LTI assignment placement in Canvas to add modules. Remove dead code, update the `README`:

https://github.com/bebraven/platform/tree/highlander-rm-ltiassignmentplacement#configure-canvas-api-access-token
Screen Shot 2020-12-11 at 12 01 21 PM

https://github.com/bebraven/platform/tree/highlander-rm-ltiassignmentplacement#does-it-work
Screen Shot 2020-12-11 at 12 01 29 PM

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have added neccessary labels
- [ ] My PR title follows the convention in the guide
- [ ] I have assigned myself to the PR
- [ ] I have requested reviews from team members
- [ ] I have created a task for reviewers on PM tool
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by rshipp — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -150,24 +166,13 @@ Setup [ngrok.io](https://dashboard.ngrok.com/) in order to expose your local dev
 1. Start an ngrok tunnel with the command: `ngrok http https://platformweb -subdomain=<insertyourname>platform`
 1. Check that you can hit your local dev env from the public internet at: `https://<insertyourname>platform.ngrok.io`
 
-#### Configure and Deploy Your Personal LTI Extension
```
[the remark this answers is not in the document]
Comment (sadleb) [reply README.md:153]: There aren’t enough emoji options in the standard reactions. Overambitious is your MO 😂

File: README.md [hunk 2]
```diff
@@ -137,20 +137,25 @@ Edit the **email** and **canvas_id** in the following to match those of your tes
 User.create email: 'some_email', canvas_id: canvas_id, admin: false, first_name: 'Test', last_name: 'Student', password: 'some_password', confirmed_at: DateTime.now
 ```
 
-#### LTI extension
-We add functionality to our [online Portal](https://braven.instructure.com) (aka Canvas) by implementing an LTI Extension. Details of what an LTI Extension is can be [found here](https://docs.google.com/document/d/1sLFnqo8-lr556EwyIUHy_jLWOwzGEkub6nFKDWPO58Y/edit?usp=sharing). In order to work on things are added to the Portal through LTI, you'll need to do the following:
```
Comment (sadleb) [inline README.md:141]: I think you should add all of this back? We just don't use the LTI selection placement anymore to insert LTI launchable resources but all of our stuff is still launched through the LTI extension and you still need the LTI Developer Key setup and SSL support and all of that.

Comment (sadleb) [reply README.md:141]: nvm, I see that you did add it back, just below. ignore this.

**Review by sadleb — APPROVED** [review]
lgtm

**Pull request merged**